components/gallery/GalleryItemAction/GalleryItemActionSection.vue

```diff
@@ -43,4 +49,46 @@ watchEffect(async () => {
 })
 </script>
 
-<style scoped></style>
+<style lang="scss" scoped>
+@import '@/styles/abstracts/variables';
+
+.gallery-item-action-section {
+  justify-content: space-between;
+  align-items: center;
+
+  &__price {
+    margin-right: 1rem;
+    font-size: 2rem;
+    &-box {
+      align-content: center;
+    }
+  }
+  @include until-widescreen {
+    flex-direction: column;
+    justify-content: flex-start;
+    align-items: flex-end;
+    &__info {
+      min-width: 10rem;
+      text-align: left;
+      > p {
+        font-size: 0.8rem;
+        margin-bottom: -0.3rem;
```

Comment:
negative margin?

Reply:
That would match the new design for mobile screens. Leave it for the visual check round.

Reply:
you could use `div` instead of `p` since it's more of a label than a paragraph
same stuff for font-size you can always use Typography helpers and it would be better to unify different font sizes in app

Reply:
I have replaced `p` tag whit `div`.
As for the font-size, it looks good that a small size on mobile and big size on web screen as in the design figma. It's also by design.

Reply:
I would prefer that you change line-height instead of negative margin

Reply:
Good idea. I have updated it.